lib/code-ribbon.js

```diff
@@ -49,18 +49,10 @@ module.exports = {
       'code-ribbon:cr-update': () => this.cr_update()
     }));
 
-    this.subscriptions.add(atom.commands.add('atom-workspace', {
-      'code-ribbon:add': () => this.add()
-    }));
-
-    this.subscriptions.add(atom.commands.add('atom-workspace', {
-      'code-ribbon:remove': () => this.remove()
-    }));
-
```

Comment:
dawwww

Reply:
lol this happens automatically now with the trim operation in the update